chain/syncer_integration_test.go

```diff
@@ -104,11 +108,16 @@ func TestLoadFork(t *testing.T) {
 	// Test that the syncer can't sync a block chained from on the right (originally shorter) chain
 	// without getting old blocks from network. i.e. the store index has been trimmed
 	// of non-heaviest chain blocks.
-	err = offlineSyncer.HandleNewTipset(ctx, newRight.Key(), "")
+	nrH, err := newRight.Height()
+	require.NoError(t, err)
+
+	err = offlineSyncer.HandleNewTipset(ctx, chain.NewPeerHeadInfo(newRight.Key(), types.Uint64(nrH), "", true))
```

Comment:
Consider reducing the repeated verbosity by adding a `headFromTip(types.TipSet)` method to do the type wrangling just once